python/src/scipp/plotting/resampling_model.py

```diff
@@ -120,22 +120,37 @@ def _call_resample(self):
             self._home = self._resampled
             self._home_params = self._resampled_params
 
+    def _make_array(self, array):
+        return array
```

Comment:
So this is used by `ResamplingCountsModel` which does not override?

Reply:
yes

Reply:
Maybe move it there then?